src/redux/books/booksSlice.js

```diff
@@ -2,7 +2,27 @@ import { createSlice } from '@reduxjs/toolkit';
 
 const bookslice = createSlice({
   name: 'book',
-  initialState: [],
+  initialState:
+  [
+    {
+      item_id: 'item1',
+      title: 'The Great Gatsby',
+      author: 'John Smith',
+      category: 'Fiction',
+    },
+    {
+      item_id: 'item2',
+      title: 'Anna Karenina',
+      author: 'Leo Tolstoy',
+      category: 'Fiction',
+    },
+    {
+      item_id: 'item3',
+      title: 'The Selfish Gene',
+      author: 'Richard Dawkins',
+      category: 'Nonfiction',
+    },
+  ],
```

Comment:
In your `removeBook` reducer, you are filtering the books on the basis of `id` but `id` prop doesn't exist on the books. Instead it is `item_id`. Kindly either change it in the data here or update your reducer and `handleRemoveBook` method.